packages/create-emotion-styled/types/test.tsx

```diff
@@ -168,13 +168,13 @@ const getEditorLogoColor = (editor: 'vscode' | 'emacs' | 'sublime') => {
     case 'sublime':
       return 'ff9800';
   }
-}
+};
 
 const StyledCompShorthandWithProps0 = createStyled.div<ShorthandProps>(props => ({
   backgroundColor: getEditorLogoColor(props.editor),
 }));
-const StyledCompShorthandWithProps1 = createStyled.section<ShorthandProps>`
-  backgroundColor: ${props => getEditorLogoColor(props.editor)};
+const StyledCompShorthandWithProps1 = createStyled.section`
+  backgroundColor: ${(props: ShorthandProps) => getEditorLogoColor(props.editor)};
```

Comment:
What's the reason for this change?

Reply:
@mitchellhamilton It cannot pass check in TS 2.3.
We should add `yarn test:typescript` to TravisCI...

Reply:
Yeah, I removed the ts tests from travis because it was crashing and I didn't have time to figure out why, if you want to fix it and add it back [here](https://github.com/emotion-js/emotion/blob/master/package.json#L9) that would be great.

Reply:
@mitchellhamilton I will add it after this one will be merged.